qt/scientific_interfaces/Indirect/IndirectFitPropertyBrowser.cpp

```diff
@@ -16,6 +16,7 @@
 #include "MantidAPI/WorkspaceGroup.h"
 
 #include "MantidQtWidgets/Common/FitOptionsBrowser.h"
+#include "MantidQtWidgets/Common/FittingMode.h"
```

Comment:
@Harrietbrown you may want to have a look at this, it changes some of the indirect fitting